src/operator/concat.cc

```diff
@@ -50,6 +55,18 @@ Operator* CreateOp<cpu>(ConcatParam param, int dtype) {
   if (enableMKLWarnGenerated())
     LOG(INFO) << MKLConcatOp<cpu, float>::getName() << " Skip MKL optimization";
 #endif
+#if MXNET_USE_MKLDNN == 1
+  if ((1 == param.dim) && (param.num_args > 1)) {
+    switch (dtype) {
+      case mshadow::kFloat32:
+        return new MKLDNNConcatOp<cpu, float>(param);
```

Comment:
Similar comment as above.

Reply:
This is missing. I will fix.

Reply:
There are several other ops that have the same problem. I omitted them for brevity. Let me know once you fixed the types.